Resources/Prototypes/_LostParadise/Datasets/dreamadverbs.yml

```
@@ -0,0 +1,139 @@
+﻿- type: dataset
+  id: dreamadverbs
+  values:
+    #ss13
```

yamllint

[warning] missing starting space in comment

(comments)